LibreNMS/Modules/Transceivers.php

```diff
@@ -41,7 +41,7 @@ class Transceivers implements Module
 
     public function dependencies(): array
     {
-        return ['ports'];
+        return ['ports', 'entity-physical'];
```

Comment:
I think I wrote the code to avoid this dependency.  It will use the DB if available, otherwise it will run snmp queries.

Reply:
OK, I'll remove it. Tests should still pass successfully If I did not mess up :)

Reply:
After removing it, it seems that comware does need this dependency to have the expected result. So we can 
- ignore it (and I revert comware tests) 
- force it for everybody 
- force it on an OS base (like force it for comware, but not for ciscoShared ?)

Reply:
removing the dependency breaks it again (at least for ASR9k), and I can't tell if the issue is with the tests processing or if it is a really issue with the code. I can't reproduce it locally. 
Any reason to avoid the dependency ? Cause adding it makes the tests more predictable, and removes redundant code anyway ...